README.md

```diff
@@ -94,6 +113,16 @@ Your Answer:
 What are the differences between calling and referencing a function? Please provide examples of each.
 
 ```text
+function someFunction(){
+  //code goes here
+}
+
+//calling the function on the console line looks like this and will return a value:
+someFunction();
+32
+
+//referencing a function looks like this on the console line and will display the contents of the function
+someFunction (){//code goes here}
```

Comment:
Please review referencing a function.  
At the console, you can show a reference to the function by simply writing the function name, without the parens.  
```
someFunction
```
Where do we commonly see a reference to a function used?